### DISCUSSION TITLE
Scores with accuracy challenge mod are not being submitted

### DISCUSSION THREAD

**Discussion by Bruno5430** [ppy/osu#22567] [Q&A] [answered]
I'm asking here as a discussion because i don't know if it's the client or the server's fault
It happens with every ruleset and with any configuration of the AC mod.

![osu_2023-02-08_22-30-48](https://user-images.githubusercontent.com/111712756/217692618-1a83ad6d-cfb9-4a3a-a742-5766e873f6a1.png)

Version:
2023.207.0

Logs:
[database.log](https://github.com/ppy/osu/files/10691787/database.log)
[input.log](https://github.com/ppy/osu/files/10691788/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10691789/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10691790/network.log)
[performance.log](https://github.com/ppy/osu/files/10691792/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/10691793/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/10691794/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/10691795/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/10691796/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/10691797/runtime.log)

**Comment by peppy** [comment, marked as answer]
This should be fixed now! Sorry about that.
